messaging-experiments.yaml

```diff
@@ -0,0 +1,34 @@
+- id: aw_pull_factor_privacy_1
```

Comment:
nit: shall we consider using capital case here?

Reply:
good question, I think we should go with whatever is standard in normandy right now (looks like lowercase)

Reply:
I take back my previous comment, this can be uppercase (or whatever we want)

Reply:
Yeah, I think Normandy's convention is using lowercase for slugs and branches, not sure about the recipe id. If it's up to us to decide,  uppercase would be easier to catch and manage.